HelpSource/Classes/Ndef.schelp

```diff
@@ -4,7 +4,9 @@ categories:: Libraries>JITLib>NodeProxy, Live Coding
 related:: Classes/ProxySpace, Classes/Tdef
 
 description::
-Reference to a proxy, forms an alternative to link::Classes/ProxySpace::. All methods are inherited from link::Classes/NodeProxy::.
+Ndef builds a synth, the internals of which can be replaced while playing. An Ndef can be started, stopped and manipulated through its key. The old synth and its replacement can automatically crossfade and the replacement time can be quantized.
```

Comment:
The class Ndef (and Tdef etc.) is an *accessor* to instances. All access runs through the class, not the instance.

See also http://quod.lib.umich.edu/i/icmc/bbp2372.2005.135?rgn=main;view=fulltext